cms/server/admin/handlers/base.py

```diff
@@ -68,7 +69,6 @@
 
 logger = logging.getLogger(__name__)
 
-
```

Comment:
And restore this empty line. Actually, are you using VS code? It should automatically re-format on save, though you probably need to install some extension (it should automatically recommend to install Ruff)